charactersheet/charactersheet/templates/character/armors.tmpl.html

```diff
@@ -28,6 +31,11 @@
           <td data-bind="click: $parent.editArmor"
               href="#"
               data-target="#viewArmor">
+            <span data-bind="css: $parent.armorEquippedLabel($data)"></span>
+          </td>
+          <td data-bind="click: $parent.editArmor"
+              href="#"
+              data-target="#viewArmor">
```

Comment:
Can we remove this `data-target`? The `editArmor` method should be the only thing handling when a modal opens. The `data-target` causes issues with the modal.